repositories/continuous_screening_repository.go

```diff
@@ -75,14 +75,31 @@ func (repo *MarbleDbRepository) GetContinuousScreeningConfigsByOrgId(
 	}
 
 	sql := NewQueryBuilder().
-		Select(dbmodels.ContinuousScreeningConfigColumnList...).
+		Select(dbmodels.SelectContinuousScreeningConfigColumnList...).
 		From(dbmodels.TABLE_CONTINUOUS_SCREENING_CONFIGS).
 		Where(squirrel.Eq{"org_id": orgId}).
 		Where(squirrel.Eq{"enabled": true})
 
 	return SqlToListOfModels(ctx, exec, sql, dbmodels.AdaptContinuousScreeningConfig)
 }
 
+func (repo *MarbleDbRepository) ListContinuousScreeningConfigs(
+	ctx context.Context,
+	exec Executor,
+) ([]models.ContinuousScreeningConfig, error) {
+	if err := validateMarbleDbExecutor(exec); err != nil {
+		return nil, err
+	}
+
+	query := NewQueryBuilder().
+		Select(dbmodels.SelectContinuousScreeningConfigColumnList...).
+		From(dbmodels.TABLE_CONTINUOUS_SCREENING_CONFIGS).
+		Where(squirrel.Eq{"enabled": true}).
+		OrderBy("id")
```

Comment:
As far as I know, we are using UUIDv4 for the `id` column, this will make the order of this list weird, where there is no understandable ordering, and something you create can be placed somewhere in the middle of the list.

Either order by something else, or use UUIv7?